sass/_Quickview.scss

```diff
@@ -9,9 +9,31 @@
     @include retina-sprite($quickview-group);
   }
 
-  .coveo-caption-for-quickview {
+  &:hover .coveo-caption-for-icon {
+    display: inline;
+  }
+  
+  .coveo-caption-for-icon {
     font-size: $font-size-smallest;
-    margin-top: -7px;
+    display: none;
+    background: $color-greyish-dark-blue;
+    color: white;
+    border-radius: 2px;
+    padding: 6px 16px;
+    position: absolute;
+    white-space: nowrap;
+    transform: translate(calc(-50% - 11px), calc(100% - 6px));
```

Comment:
Fixed it on IE11 and Edge, using multiple transforms instead of a `calc`
See updated file

![image](https://cloud.githubusercontent.com/assets/2488155/26067037/cd589994-3966-11e7-84ad-4603e251d081.png)